tests/zenko_e2e/backbeat/BackbeatAPIUtility.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const { makeGETRequest, makePOSTRequest, getResponseBody } =
+    require('./utils/request');
+
+class BackbeatAPIUtility {
+    constructor() {}
+
+    getReplicationStatus(locationName, cb) {
+        let path = '/_/backbeat/api/crr/status';
+        if (locationName) {
+            path = `${path}/${locationName}`;
+        }
+        makeGETRequest(path, (err, res) => {
+            assert.ifError(err);
```

Comment:
Since we are asserting the error in the `it` statement, maybe it's cleaner to return the errors from the helper. When asserting in both places, the test suite assertion isn't needed. Also, then you can just pass in the `cb` to `getResponseBody`. 😄

Reply:
I see, makes sense, will fix asap